index.html

```diff
@@ -44,53 +44,91 @@ <h2>Who</h2>
       We are cybersecurity enthusiasts. If you share the idea of global collaboration to combat threats, you are very welcome to join the next sprint.
       <br><br>
       <h2>When</h2>
-      The next two weeks-long sprint will start <a href="./sprints/sprint_2.html">October 5, 2020</a>.
+      The second two weeks-long sprint ended in <a href="./sprints/sprint_2.html">October 2020</a>.
```

Comment:
It's a good idea to mention the previous sprint!
Let's use the "previous" and "next" words (not 3rd/2nd), as readers don't really need to know the number here.

> The third two weeks-long sprint will start <a href="./sprints/sprint_3.html">Fall 2021</a>.

I would reuse [our former version](https://github.com/oscd-initiative/oscd/blob/4612cf1abb139a55cf76155864ad48a81a8a8865/index.html#L41):

> The next sprint is planned for Winter 2021. Details will be provided on our <a href="https://twitter.com/oscd_initiative">twitter</a> soon.

Reply:
I agree, thanks!

Reply:
https://github.com/oscd-initiative/oscd/pull/4/commits/9e639dffc3c8efb5ed756222da75a1408e508a8c

Reply:
Done